doc/Language/regexes.pod6

```diff
@@ -1934,7 +1934,7 @@ sub show_captures( Match $m ){
     say $result_split;
 }
 
-$string ~~ /(.+)(SQL) (.+) $1 (.+) { show_captures( $/, $string );  }/;
+$string ~~ /(.+)(SQL) (.+) $1 (.+) { show_captures( $/ );  }/;
```

Comment:
Two advice here.

1. use :preamble<my $string;> after `=begin code` to fit xtest.

2. rename `show_captures` to `show-captures` to fit naming convention.

Anyway, you can be brave to commit directly. We just trust our contributors and it's ok there is something not that good. Your caution is appreciated for me though.